.github/workflows/publish-release-container.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and Publish container for new releases
+
+on:
+  release:
+    types:
+      - published
```

Comment:
this will happen only when you release it, is it all right?

Reply:
yes, i think is the best way to go to publish the image when a new tag is created.